<?xml version="1.0" ?>
<launch>

  <!-- By default, we are not in debug mode -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --args" />

  <node name="plot_dataset" pkg="moveit_benchmark_suite" type="plot_dataset" respawn="false" required="true" output="screen" launch-prefix="$(arg launch_prefix)">

    <rosparam param="gnuplot_config">
      files:
        - /home/captain-yoshi/.ros/db/mp1.yaml
      xtick_filters:
        - config/scene/
      legend_filters:
        - config/collision_detector/
      metrics:
        - time: boxplot
        - correct: boxplot
      options:
          n_row: 1
          n_col: 2
          output_script: true
    </rosparam>

  </node>
</launch>
